legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabToken.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.gitlab.auth;
+
+import org.gitlab4j.api.Constants.TokenType;
+
+class GitLabToken
```

Comment:
Maybe we should add a factory method that takes a TokenType and a String. That way, we don't have to worry about which factory method to call based on the token type we have.